runtime/src/lib.rs

```diff
@@ -389,6 +398,190 @@ impl anchor::Trait for Runtime {
+    /// Only council members can vote
+    type VoterOrigin = CouncilMember;
+}
+
+impl pallet_democracy::Trait for Runtime {
+    type Proposal = Call;
+    type Event = Event;
+    type Currency = Balances;
+    type EnactmentPeriod = EnactmentPeriod;
+    type LaunchPeriod = LaunchPeriod;
+    type VotingPeriod = VotingPeriod;
+    type CooloffPeriod = CooloffPeriod;
+    type MinimumDeposit = MinimumDeposit;
+    /// Only specified to compile, not used however.
+    type ExternalOrigin = CouncilMember;
+    type ExternalMajorityOrigin = CouncilMember;
+    /// Only specified to compile, not used however.
+    type ExternalDefaultOrigin = RootOrMoreThanHalfCouncil;
+    /// Two thirds of the technical committee can have an ExternalMajority/ExternalDefault vote
+    /// be tabled immediately and with a shorter voting/enactment period.
+    type FastTrackOrigin = EnsureOneOf<
+        AccountId,
+        pallet_collective::EnsureProportionAtLeast<_2, _3, AccountId, TechnicalCollective>,
+        EnsureRoot<AccountId>,
+    >;
+    type InstantOrigin = EnsureRoot<AccountId>;
+    type InstantAllowed = ();
+    type FastTrackVotingPeriod = FastTrackVotingPeriod;
+    type CancellationOrigin = RootOrMoreThanHalfCouncil;
+    /// Any council member can cancel a public proposal
+    type CancelProposalOrigin = CouncilMember;
+    type PreimageByteDeposit = PreimageByteDeposit;
+    /// Slashes are handled by Democracy
+    type Slash = Democracy;
+    type OperationalPreimageOrigin = CouncilMember;
+    type VetoOrigin = pallet_collective::EnsureMember<AccountId, TechnicalCollective>;
+    type Scheduler = Scheduler;
+    type PalletsOrigin = OriginCaller;
+    type MaxVotes = MaxVotes;
+    type MaxProposals = MaxProposals;
+    type WeightInfo = ();
+}
+
+/// Filter to disallow access to certain modules. Make sure that any modules added to `Runtime` (with `construct_runtime!`)
+/// and intended to called publicly should be added to this filter.
+pub struct BaseFilter;
+
+impl Filter<Call> for BaseFilter {
+    fn filter(call: &Call) -> bool {
+        match call {
+            // These modules are all allowed to be called by transactions:
+            Call::Democracy(_)
+            | Call::Council(_)
+            | Call::TechnicalCommittee(_)
+            | Call::CouncilMembership(_)
+            | Call::TechnicalCommitteeMembership(_)
+            | Call::System(_)
+            | Call::Scheduler(_)
+            | Call::Timestamp(_)
+            | Call::Balances(_)
+            | Call::Authorship(_)
+            | Call::Session(_)
+            | Call::Grandpa(_)
+            | Call::Utility(_)
+            | Call::PoAModule(_)
+            | Call::DIDModule(_)
+            | Call::Revoke(_)
+            | Call::BlobStore(_)
+            | Call::Anchor(_)
+            | Call::Master(_)
+            | Call::Sudo(_)
+            | Call::MigrationModule(_)
+            | Call::RandomnessCollectiveFlip(_) => true,
+            _ => false,
```

Comment:
Please match exhaustively on `Call`. By that I mean don't use the `_` as a fall-through.

```suggestion
            Call::ForkedDemocracy(_) => false,
```

Reply:
Made the change. Is that ok?